src/hooks/useNotifications.tsx

```diff
@@ -5,35 +5,35 @@ import { TransactionReceipt } from "web3-core";
 
 export const useNotifications = () => {
   const { enqueueSnackbar, closeSnackbar } = useSnackbar();
-
   const notify = (
     message: string,
     variant: VariantType,
-    txHash?: TransactionReceipt
+    txHash?: TransactionReceipt,
+    chainId?: number
   ) => {
     enqueueSnackbar(message, {
       variant,
       autoHideDuration: 6000,
       anchorOrigin: {
         vertical: "bottom",
-        horizontal: "right",
+        horizontal: "left",
       },
-      action: (key) => {
+      action: (key) => (
         <Fragment>
           {txHash && (
             <Button
-              href={`https://etherscan.io/tx/${txHash.blockHash}`}
+              href={`https://${chainId === 4 ? 'rinkeby.' : ''}etherscan.io/tx/${txHash.transactionHash}`}
```

Comment:
I wrote some convenience functions for most URLs that get repeated here: https://github.com/defiat-crypto/defiat-frontend-v2/blob/main/src/utils/address.ts#L17-L21